enterprise/tailnet/pgcoord.go

```diff
@@ -1140,7 +1146,11 @@ func (h *heartbeats) recvBeat(id uuid.UUID) {
 		h.logger.Debug(h.ctx, "set initial heartbeat timeout")
 		return
 	}
+	h.resetExpiryTimer()
+}
 
+func (h *heartbeats) resetExpiryTimer() {
```

Comment:
nit: Does `resetExpiryTimer` requires a lock? If so, maybe rename the method to emphasize it (`..WithLock`).

Reply:
it does require the lock.  I'll rename